core/server/master/src/main/java/alluxio/master/file/meta/TtlBucket.java

```diff
@@ -78,38 +81,58 @@ public static long getTtlIntervalMs() {
   }
 
   /**
-   * @return the set of all inodes in the bucket backed by the internal set, changes made to the
-   *         returned set will be shown in the internal set, and vice versa
+   * @return the set of all inodes ids in the bucket backed by the internal set,
+   * changes made to the returned set will be shown in the internal set, and vice versa.
    */
-  public Collection<Inode> getInodes() {
-    return mInodes.values();
+  public Collection<Long> getInodeIds() {
+    return mInodeToRetryMap.keySet();
```

Comment:
nit: `return Collections.unmodifiableSet()`?

Reply:
not a nit, thanks.